src/doppler/main.go

```diff
@@ -198,36 +195,31 @@ func main() {
 	// Post Start
 	//------------------------------
 
-	for {
-		select {
-		case <-dumpChan:
-			signalmanager.DumpGoRoutine()
-		case <-killChan:
-			log.Print("Shutting down")
-
-			stopped := make(chan bool)
-			legacyStopped := make(chan bool)
-			releaseNodeChan <- stopped
-			legacyReleaseNodeChan <- legacyStopped
-
-			stop(
-				errChan,
-				wg,
-				openFileMonitor,
-				uptimeMonitor,
-				appStoreWatcher,
-				udpListener,
-				sinkManager,
-				websocketServer,
-				storeAdapter,
-			)
-
-			<-stopped
-			<-legacyStopped
-
-			return
-		}
-	}
+	killChan := signalmanager.RegisterKillSignalChannel()
```

Comment:
Sure. Do we even need that function. Maybe we can just pull the functionality in.